src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -133,7 +135,7 @@ private function compileRoutes(RouteCollection $routes, $supportsRedirections)
         foreach ($groups as $collection) {
             if (null !== $regex = $collection->getAttribute('host_regex')) {
                 if (!$fetchedHost) {
-                    $code .= "        \$host = \$this->context->getHost();\n\n";
```

Comment:
This removes a reference lookup, probably hardly any impact, but in optimised code, every cycle counts, right?